pkgs/os-specific/linux/kernel/linux-6.2.nix

```
@@ -1,4 +1,4 @@
-{ lib, buildPackages, fetchurl, perl, buildLinux, nixosTests, ... } @ args:
+{ lib, buildPackages, fetchurl, perl, buildLinux, nixosTests, fetchpatch, ... } @ args:
```

```suggestion
{ lib, buildPackages, fetchurl, perl, buildLinux, nixosTests, ... } @ args:
```
This isn't needed, right?

---

Correct